### ISSUE TITLE
gpg expire "Wrong secret key" when renewing subkeys

### ISSUE THREAD

**Issue by Sebazzz** [drduh/YubiKey-Guide#375]
Running the procedure for renewing the subkeys on my primary computer succeeded. I'm now doing it on my secondary computer (same Yubikeys), but fail to complete it:

```
gpg> expire
Are you sure you want to change the expiration time for multiple subkeys? (y/N) y
Please specify how long the key should be valid.
         0 = key does not expire
      <n>  = key expires in n days
      <n>w = key expires in n weeks
      <n>m = key expires in n months
      <n>y = key expires in n years
Key is valid for? (0) 3y
Key expires at 6-3-2026 08:12:38 W. Europe Standard Time
Is this correct? (y/N) y
gpg: signing failed: Foute geheime sleutel
gpg: make_keysig_packet failed: Foute geheime sleutel
```

I'm not sure what I'm doing wrong. I'm following the same procedure as on my primary computer. It seems similar [to this issue](https://unix.stackexchange.com/q/393162):

> I was using an offline master key and used subkeys for signing and decryption. This was displayed by the # in front of my master key.

This is also the case with me.

Are there perhaps any files I can copy to remedy the problem?

**Comment by sza-1** [comment]
> Are there perhaps any files I can copy to remedy the problem?

Check `gpg --card-status` and `gpg -K`. The latter one should shows `>` for sec key and also some `Card serial no.` if GPG detected your card.

**Comment by Sebazzz** [comment]
I got it sorted now, but I'm doing a [denvercoder9](https://xkcd.com/979/) now, because I'm not sure how I got it working.

I suspect that with the Yubikey locked, the procedure doesn't work. _I think_ I went partially through the procedure for generating a new key, and then tried again. I'm also not sure which files were eventually changed in `%APPDATA%\gnupg` as this should only be about the *public* key right?
